packages/pwa-kit-react-sdk/src/webpack/config.js

```diff
@@ -187,7 +187,16 @@ const common = {
         }),
 
         new CopyPlugin({
-            patterns: [{from: 'app/static/', to: 'static/'}]
+            patterns: [
+                {from: 'app/static/', to: 'static/'},
+                {
+                    from: 'config/',
+                    to: 'config/',
```

Comment:
nit, shall we follow the existing folder naming pattern and rename the folder to `configs`?

Reply:
I think config "singular" is an established pattern. Check your home folder for a `.config` folder. There is probably one there and it hold configurations for all sorts of things.